radicle-node/src/service.rs

```diff
@@ -795,6 +824,7 @@ where
                             self.clock.timestamp(),
                             &self.storage,
                             &self.signer,
+                            self.filter.clone(),
```

Comment:
How big can the filter be?  Will it be small enough to be supported by 'radicle-node::wire'?

Reply:
There are three sizes, the largest is 16KB.